src/flamenco/vm/fd_vm.c

```diff
@@ -201,6 +203,13 @@ fd_vm_validate( fd_vm_t const * vm ) {
       break;
     }
 
+    case FD_CHECK_CALLX: {
+      /* The register number to read is stored in the immediate.
+         https://github.com/solana-labs/rbpf/blob/v0.8.1/src/verifier.rs#L218 */
+      if( FD_UNLIKELY( instr.imm>9 ) ) return FD_VM_ERR_INVALID_REG;
```

Comment:
We need to specifically feature gate disallowing register 10 (`reject_callx_r10`). Registers > 10 are always disallowed.